client-app/shared/account/composables/useUser.ts

```diff
@@ -112,6 +103,22 @@ export default () => {
     }
   }
 
+  async function signMeOut(): Promise<void> {
+    try {
```

Comment:
Should we prevent multiple calling by adding `if (loading.value) return;` before `try..catch` block?

Reply:
Good suggestion, we should prevend duplicate requests.

Reply:
I think we should up this concern to a high level. There is just an API request layer.